guidelines/index.html

```diff
@@ -1586,23 +1586,23 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
  </section>
 
             
-            <section class="guideline" id="target-size-minimum"><div class="header-wrapper"><h4 id="x2-5-8-target-size-minimum"><bdi class="secno">Success Criterion 2.5.8 </bdi>Target Size (Minimum)</h4><a class="self-link" href="#target-size-minimum" aria-label="Permalink for Section 2.5.8"></a></div>
+            <section class="guideline" id="target-size-minimum"><div class="header-wrapper"><h4 id="x2-5-8-target-size-minimum"><bdi class="secno">達成基準 2.5.8 </bdi>ターゲットのサイズ (最低限)</h4><a class="self-link" href="#target-size-minimum" aria-label="Permalink for Section 2.5.8"></a></div>
    
     
     
     <div class="doclinks"><a href="https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html">Understanding Target Size (Minimum)</a> <span class="screenreader">|</span> <br><a href="https://www.w3.org/WAI/WCAG22/quickref/#target-size-minimum">How to Meet Target Size (Minimum)</a></div><p class="conformance-level">(レベル AA)</p>
 	<p class="change">[New]</p>
     
-    <p>The size of the <a href="#dfn-targets" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-targets-2" title="ユーザインタフェース コンポーネントのインタラクティブな領域のような、ポインタアクションを受け入れるディスプレイの領域">target</a> for <a href="#dfn-pointer-inputs" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-pointer-inputs-2" title="マウス、ペン、タッチ接触のように、画面上の特定の座標 (又は複数の座標群) をターゲットにできるデバイスからの入力。">pointer inputs</a> is at least 24 by 24 <a href="#dfn-css-pixels" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-css-pixels-5" title="約 0.0213 度の視野角。">CSS pixels</a>, except where:</p>
+    <p><a href="#dfn-pointer-inputs" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-pointer-inputs-2" title="マウス、ペン、タッチ接触のように、画面上の特定の座標 (又は複数の座標群) をターゲットにできるデバイスからの入力。">ポインタ入力</a>の<a href="#dfn-targets" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-targets-2" title="ユーザインタフェース コンポーネントのインタラクティブな領域のような、ポインタアクションを受け入れるディスプレイの領域">ターゲット</a>のサイズが 24 × 24 <a href="#dfn-css-pixels" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-css-pixels-5" title="約 0.0213 度の視野角。">CSS ピクセル</a>以上である。ただし、以下の場合は例外とする。</p>
```

Comment:
コントラスト比(最低限)を見ながら変更してみました。
```suggestion
    <p><a href="#dfn-pointer-inputs" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-pointer-inputs-2" title="マウス、ペン、タッチ接触のように、画面上の特定の座標 (又は複数の座標群) をターゲットにできるデバイスからの入力。">ポインタ入力</a>の<a href="#dfn-targets" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-targets-2" title="ユーザインタフェース コンポーネントのインタラクティブな領域のような、ポインタアクションを受け入れるディスプレイの領域">ターゲット</a>のサイズは、少なくとも<a href="#dfn-css-pixels" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-css-pixels-5" title="約 0.0213 度の視野角。">CSS ピクセル</a>である。ただし、次の場合は除く:</p>
```

Reply:
既存の、達成基準 2.5.5 の訳も、これに合わせて修正する必要があります。

> ポインタ入力のターゲットのサイズが 44 × 44 CSS ピクセル以上である。ただし、以下の場合は例外とする。

↓

> ポインタ入力のターゲットのサイズは、少なくとも 44 × 44 CSS ピクセルである。ただし、ただし、次の場合は除く:

[85c4717](https://github.com/waic/wcag22/pull/38/commits/85c4717ba903d713d57ad2e48837e85a279ab410) でコミットしました。